ckanext/xloader/templates/package/resource_read.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+{% block action_manage_inner %}
```

Comment:
This is another "only in master" feature. On any released version, it will not take effect as this block does not exist in the parent template.

Reply:
added max 2.10 version check in other block now.